violet/runner.py

```diff
@@ -271,28 +276,74 @@ def _exec_std_import(self, stmt):
 		# print(stmt)
 		form = stmt.from_module
 		name = form.transform_to_string()
+		vi_file_name = os.path.join("violet", *name.split(".")) + ".vi"
+		is_vi_file = os.path.exists(vi_file_name)
+		if is_vi_file:
+			try:
+				module = Runner.open(vi_file_name)
+				module.interpret()
+			except Exception as e:
+				raise StatementError(stmt, f'module {stmt.from_module.name!r} does not exist')
+
+			else:
+				for identifier in stmt.importing:
+					if identifier.name == "*":
+						vars = module.get_current_scope().vars
+						scope = self.get_current_scope()
+						for ident, value in vars.items():
+							scope.set_var(ident, value)
+
+						break
+
+					else:
+						try:
+							var = module.get_current_scope().get_var(identifier)
+						except Exception as e:
+							raise StatementError(stmt, f'failed to import {identifier.name!r} from {name}')
+						else:
+							self.get_current_scope().set_var(identifier, var)
 
-		try:
-			# print("importing violet."+name)
-			module = importlib.import_module('violet.'+name)
-		except ImportError:
-			raise StatementError(stmt, f'module {name!r} does not exist')
 		else:
-			for iport in stmt.importing:
-				if not hasattr(module, iport.name):
-					raise StatementError(stmt, f'failed to import {iport.name!r} from {name!r}')
-				else:
-					# print("imported", iport)
-					self.get_current_scope().set_var(iport, getattr(module, iport.name))
-		# print(self.get_current_scope())
+			try:
+				module = importlib.import_module('violet.'+name)
+			except ImportError:
+				raise StatementError(stmt, f'module {name!r} does not exist')
+			else:
+				for iport in stmt.importing:
+					if iport.name == "*":
+						if not hasattr(module, "__all__"):
+							importables = (x for x in dir(module) if not x.startswith("_"))
```

Comment:
I think the filter should be changed to not support values with numbers; ie
```suggestion
							importables = (x for x in dir(module) if x.isalpha() or not x.startswith("__"))
```